data/dataToSeed.js

```diff
@@ -0,0 +1,148 @@
+
```

Comment:
@VALE46GP Are you using this file at all?

Reply:
actually no.. i'll remove it